.github/actions/download-pr-data-artifact/action.yaml

```diff
@@ -0,0 +1,26 @@
+name: "Save PR number as artifact"
+description: "Save PR number as artifact"
```

Comment:
Can you expand on this? Someone looking at this should be able to understand what it does and why it's important.